### ISSUE TITLE
Accessibility issues in mkdocs materials

### ISSUE THREAD

**Issue by Page-MS** [squidfunk/mkdocs-material#8269]
### Context

Mkdocs material pages currently fails to meet accessibility standards. This means that all webpages that use it will also show the same accessibility issues and therefore be  difficult to navigate for users with low vision.

### Bug description

Testing various Mkdocs materials documentation (including the official one) with Wave, a tool to detect accessibility errors in web page, we see various recurrent issues.

![Image](https://github.com/user-attachments/assets/ed205d16-dcd7-4e5f-be27-2666f03a1fd6)

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
-https://wave.webaim.org/report#/https://mrkeo.github.io/

### Reproduction

See your own documentation

### Steps to reproduce

1-Simply put the web address inside wave or go to this link: https://wave.webaim.org/report#/https://mrkeo.github.io/
2-See the report

### Browser

Firefox

### Before submitting

- [x] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [x] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [x] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [ ] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. Closing for the reasons stated in [our bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/#incomplete-issues-and-duplicates):

> Limitations of automated tools: While we believe in the value and efficiency that automated tools bring to identifying potential issues (such as those identified by Lighthouse, Accessibility tools, and others), simply submitting an issue generated by these tools does not constitute a complete bug report. These tools sometimes produce verbose outputs and may include false positives, which necessitate a critical evaluation. You are of course welcome to attach generated reports to your issue. However, this does not substitute the requirement for a minimal reproduction or a thorough discussion of the findings. We reserve the right to mark these issues as incomplete and close them. This practice ensures that we are addressing genuine concerns with precision and clarity, rather than navigating through extensive automated outputs.

If you can show that the labels (which are necessary so Material for MkDocs works without JavaScript) are actually a problem in the daily use of a screen reader, we're happy to accept PRs. In any other case, please don't blindly trust those tools.

Also, duplicate of:

- #6837
- #4006
- likely others

Also, for a deeper explanation why we choose to us labels, see:

- https://github.com/squidfunk/mkdocs-material/issues/4006#issuecomment-1150299623

**Comment by Page-MS** [comment]
Thank you for this thorough answer and the very useful open source project !
